### DISCUSSION TITLE
Set Console WIndow net to the blocks

### DISCUSSION THREAD

**Discussion by andrewg78** [BIPES/BIPES#90] [Q&A] [answered]
How to achieve this window arrangement (console next to the blocks) ?
https://github.com/BIPES/BIPES/issues/79#issuecomment-1001673846

**Comment by rafaelaroca** [comment, marked as answer]
This is easy! :-)

Click with the left mouse button on the first tab you want to show: console, for example.
Next, click on the second tab you want to show side by side, with the right mouse button! 

You can do it with any pair of windows!

![image](https://user-images.githubusercontent.com/31751413/147587555-ca82a6c9-9d12-4229-8105-7eb42cc685ae.png)

![image](https://user-images.githubusercontent.com/31751413/147587571-4b8d6490-f5e0-42b4-af47-c4c5c758c19f.png)

**Comment by rafaelaroca** [comment]
Here are some more details of this feature, created by @JorgeGMarques :100: 
https://github.com/BIPES/BIPES/discussions/76

**Comment by andrewg78** [comment]
This is really cool. It should not be hidden from users.
Maybe as  enhancement, we can have a new icon on the tool bar(HELP) ??
![icons](https://user-images.githubusercontent.com/2042344/147643044-c60b5033-10ae-48ce-b35c-3b00ace84899.png)
It could open a small modal window with all the keyboard and mouse  shortcuts described.

**Reply by rafaelaroca** [reply]
Yes, this modal window with shortcuts would be nice! :-)
